instagram/src/App.js

```diff
@@ -1,44 +1,28 @@
 import React, { Component } from 'react';
-import dummyData from './dummy-data';
-import './App.css';
-import PostContainer from './components/PostContainer/PostContainer'
-import SearchBar from './components/SearchBar/SearchBar'
+import PostsPage from './components/PostContainer/PostsPage'
+import Login from './components/Authentication/LogIn';
+import Authenticate from './components/Authentication/Authenticate';
 
+import './App.css';
 class App extends Component {
   constructor() {
     super();
-    this.state ={
-      data: dummyData,
-      filteredUsers: [],
-      searchInput: '',
-    }
-  }
-
-  changeHandler = e => {
-    this.setState({
-      [e.target.name]: e.target.value
-    })
+    this.state = {}
   }
 
-  searchHandler = e => {
-    this.changeHandler(e)
-    this.setState((prevState) => {
-      const filteredUsersArray =  prevState.data.filter(post => post.username.includes(prevState.searchInput))
-      return {filteredUsers: filteredUsersArray}
-    })
-  }
 
   
+  
    render() {
      console.log(this.state.data)
     return (
       <div className="App">
-        <SearchBar searchHandler={this.searchHandler} searchInput={this.state.searchInput}/>
-        <PostContainer data={this.state.filteredUsers.length > 0 ? this.state.filteredUsers : this.state.data}/>
+      <PostsPage />
+        
       </div>
       
     );
   }
 }
 
-export default App;
+export default Authenticate(App)(Login);
```

Comment:
See this is so interesting, because there are so many ways to do this. This is completely different than how I did it, but it still works. Good job!